lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -294,16 +301,20 @@ public function render_cdn_url( $args = array() ) {
 	 * Render form field and description for the `link_auth0_users` option.
 	 * IMPORTANT: Internal callback use only, do not call this function directly!
 	 *
+	 * TODO: Deprecate
+	 *
 	 * @param array $args - callback args passed in from add_settings_field().
 	 *
 	 * @see WP_Auth0_Admin_Generic::init_option_section()
 	 * @see add_settings_field()
+	 *
+	 * @codeCoverageIgnore - To be deprecated
 	 */
 	public function render_link_auth0_users( $args = array() ) {
-		$this->render_switch( $args['label_for'], $args['opt_name'] );
 		$this->render_field_description(
-			__( 'Link accounts with the same verified e-mail address. ', 'wp-auth0' ) .
-			__( 'See the "Require Verified Email" setting above for more information on email verification', 'wp-auth0' )
+			__( 'This feature may currently be active. ', 'wp-auth0' ) .
```

Comment:
"may currently be active" because this setting does not (and did not before) change if the rule was changed in the dashboard .